packages/avatar-bundle/src/index.js

```diff
@@ -0,0 +1,3 @@
+export { default } from "./AvatarBundle";
+export { spacings, AVAILABLE_SPACING } from "./spacings";
+export { sizes } from "@hig/avatar";
```

Comment:
I think exporting `sizes` is unnecessary since `@hig/avatar` is a dependency users can import it from that package.

Reply:
That's a good point.